src/openms_gui/source/VISUAL/Spectrum1DCanvas.cpp

```diff
@@ -86,7 +86,7 @@ namespace OpenMS
     //Parameter handling
     defaults_.setValue("highlighted_peak_color", "#ff0000", "Highlighted peak color.");
     defaults_.setValue("icon_color", "#000000", "Peak icon color.");
-    defaults_.setValue("peak_color", "#000000", "Peak color.");
+    defaults_.setValue("peak_color", "#0000ff", "Peak color.");
     defaults_.setValue("annotation_color", "#000055", "Annotation color.");
```

Comment:
this default annotation color should be the same as the case 0 color in line 1072, right?